doozerlib/repos.py

```diff
@@ -188,72 +192,26 @@ def conf_section(self, repotype, arch=ARCH_X86_64, enabled=None, section_name=No
 
         return result
 
-    # Map arch to compatible arches
```

Comment:
so much being removed! :star_struck:

Reply:
Yeah repoquery command needs them otherwise it will not list all supported arches. It is not needed any more since because this PR reads repodata directly.